src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Unix.cs

```diff
@@ -24,7 +24,7 @@ public void FileSystemWatcher_Unix_DoesNotLeak()
             Interop.Sys.GetRLimit(Interop.Sys.RlimitResources.RLIMIT_NOFILE, out Interop.Sys.RLimit limits);
             _output.WriteLine("File descriptor limit is {0}", limits.CurrentLimit);
 
-            if (limits.CurrentLimit > 50_000)
+            if (limits.CurrentLimit > 10_000)
```

Comment:
It's odd that we're not running the test at all if the system is configured with a high limit.  Should we instead do:
```C#
int limit = Math.Min(limits.CurrentLimit, 10_000);
```
and then use `limit` in the subsequent test?

Reply:
only reason for the check is performance. The 50K loop took ~8 minutes on my computer and I've seen it fail in some containers. 
Now, if we run the loop less than the limit, it would pass even if we leak descriptor. So I don't see value in that. Maybe I should add some comment...?

Reply:
With the change in limit, is the test even running?

Reply:
I did more testing and took this change out. I'll make separate change to address  FileSystemWatcher_Unix_DoesNotLeak.